pkg/proto/pbgo/core/api.pb.gw.go

```diff
@@ -531,7 +531,7 @@ func RegisterAgentSecureHandlerServer(ctx context.Context, mux *runtime.ServeMux
 // RegisterAgentHandlerFromEndpoint is same as RegisterAgentHandler but
 // automatically dials to "endpoint" and closes the connection when "ctx" gets done.
 func RegisterAgentHandlerFromEndpoint(ctx context.Context, mux *runtime.ServeMux, endpoint string, opts []grpc.DialOption) (err error) {
-	conn, err := grpc.Dial(endpoint, opts...)
+	conn, err := grpc.Dial(endpoint, opts...) //nolint:staticcheck // TODO (ASC) fix grpc.DialContext is deprecated
```

Comment:
These are generated files, so not sure we should be hand-modifying them.

Reply:
Fair enough, removed.